src/engine/renderer/gl_shader.cpp

```diff
@@ -219,6 +219,27 @@ GLShaderManager::~GLShaderManager()
 
 void GLShaderManager::freeAll()
 {
+	for ( GLShader* &shader : _shaders )
+	{
+		shader->_vertexShaderText.clear();
```

Comment:
Why is this required? Shouldn't this be properly free'd by the std::string deconstructor?

Reply:
I don't know, even calling `clear` without calling `shrink_to_fit` doesn't stop the leak…

The leaked data seems to be the return of `GetShaderText()` in `BuildGPUShaderText()`.

Such string is a concatenation of multiple strings allocated with `reserve`, plus the return of `GetShaderText()`. That's all I know.